schema-loader/build.gradle

```diff
@@ -118,59 +118,55 @@ shadowJar {
     mergeServiceFiles()
 }
 
-task shadowJarUnversioning(type: Copy) {
-    from shadowJar
-    into '.'
-    rename { String fileName ->
-        fileName.replace("-$project.version", "")
-    }
-}
-
 task integrationTestCassandra(type: Test) {
-    dependsOn(shadowJarUnversioning)
+    dependsOn shadowJar
     description = 'Runs the schema loader integration tests for Cassandra.'
     group = 'verification'
     testClassesDirs = sourceSets.integrationTestCassandra.output.classesDirs
     classpath = sourceSets.integrationTestCassandra.runtimeClasspath
     outputs.upToDateWhen { false }  // ensures integration tests are run every time when called
     options {
         systemProperties(System.getProperties())
+        systemProperty("scalardb.schemaloader.jar_path", "${project.buildDir}/libs/${archivesBaseName}-${project.version}.jar")
```

Comment:
Rather than renaming the jar file, we can use the jar file under the build dir.

Reply:
Is there a way to set this property visible for all the tasks? Then we don't have to rewrite it in each one

Reply:
I'm not sure the way. Instead, I added a variable for the fat jar path and made use of it for the tasks.